openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/OpenShiftRouteLocator.java

```diff
@@ -52,7 +52,7 @@ private CubeOpenShiftConfiguration getCubeOpenShiftConfiguration() {
         return config;
     }
 
-    private URL createUrlFromRoute(Route route) {
+    static URL createUrlFromRoute(Route route) {
```

Comment:
I am not sure do we really need to make it static?

Reply:
I need to access it in OpenShiftAssistant, if I don't make it static, I'll have to create an instance of `OpenShiftRouteLocator` class.